doc/tutorials/enable_ptm.rst

```diff
@@ -0,0 +1,67 @@
+.. _enable-ptm:
+
+Enable PCIe Precision Time Management
+#####################################
+
+The PCI Express (PCIe) specification defines a Precision Time Measurement (PTM)
+mechanism that lets you coordinate and synchronize events across multiple PCI
+components within the same system with very fine time precision.
+
+ACRN adds PCIe root port emulation in the hypervisor to support the PTM feature
+and emulates a simple PTM hierarchy.  ACRN enables PTM in a Guest VM if the user
+sets the ``enable_ptm`` option when passing through a device to a post-launched
+VM. When the user enables PTM, the passthru device is connected to a virtual
+root port instead of the host bridge as it normally would.
+
+Here is an example launch script that enables PTM on a passthru Ethernet card:
+
+.. code-block:: bash
+   :emphasize-lines: 5
+
+   acrn-dm -A -m $mem_size -s 0:0,hostbridge \
+      -s 3,virtio-blk,uos-test.img \
+      -s 4,virtio-net,tap0 \
+      -s 5,virtio-console,@stdio:stdio_port \
+      -s 6,passthru,a9/00/0,enable_ptm \
```

Comment:
To enable PTM on guest, I suppose we need: 1) add <PTM>y<PTM> in config xml; 2) add enable_ptm in launch script.
Should we add config xml description/guide for PTM here?

Reply:
Good catch.  Yes.  We should include the change to config xml (scenario file for post launch vm)
Example:
    <PTM>y</PTM>

Reply:
@yantingj @rliu88 Can you provide me with the text for the changes we need to make here?

Reply:
@dbkinder, in industry.xml for tgl-rvp, please refer to: 
<vm id="1">
    <PTM>y</PTM>
</vm>